cmd/guacgql/cmd/server.go

```diff
@@ -111,16 +113,33 @@ func startServer(cmd *cobra.Command) {
 
 func validateFlags() error {
 	if flags.backend != neo4js &&
-		flags.backend != inmems && flags.backend != arango && flags.backend != neptune {
+		flags.backend != inmems && flags.backend != arango {
```

Comment:
I think we still need to keep `neptune` here?

Reply:
Oh definitely, that's accidentally got dropped in the merge, will fix.